project/src/pages/property-page/near-places-list/near-places-list.tsx

```diff
@@ -1,7 +1,12 @@
 import PlaceCard from '../../../components/layout/place-card/place-card';
 import { IOffer } from '../../../types/interfaces/offer.interface';
 
-export function NearPlacesList(props: {nearData: IOffer[]}) {
+type NearPlacesListProps = {
+  nearData: IOffer[];
+  onPlaceCardHover: (placeCardName: string) => void;
+}
+
+export function NearPlacesList(props: NearPlacesListProps) {
```

Comment:
можно сразу развернуть `props`
```suggestion
export function NearPlacesList({onPlaceCardHover}: NearPlacesListProps) {
```

Reply:
Это больше похоже на дело привычки ))